Makefile

```diff
@@ -142,3 +146,12 @@ stop-kind: ## Stop the local kind cluster
 .PHONY: log
 log: ## View manager logs
 	kubectl logs deploy/mongodb-atlas-kubernetes-controller-manager manager -n mongodb-atlas-kubernetes-system -f
+
+.PHONY: golangci-lint
+golangci-lint: ## Find golangci-lint or download it if necessary
```

Comment:
I think this one will be unnecessary if the comment above is addressed. The golint will be downloaded automatically then.